### PR TITLE
update for tutor lms 2.1x

### PR DESCRIPTION
**Pull request by wprashed**

### CODE REVIEW

File: templates/course/reviews.php [hunk 1]
```diff
@@ -145,24 +148,26 @@
 </div>
 
 <?php if ( $is_enrolled ) : ?>
-	<div class="tutor-course-enrolled-review-wrap tutor-mt-16">
+	<div class="tutor-course-enrolled-review-wrap tutor-pt-16">
 		<div class="tutor-write-review-form" style="display: none;">
 			<form method="post">
 				<div class="tutor-star-rating-container">
-					<input type="hidden" name="course_id" value="<?php echo esc_attr( $course_id ); ?>"/>
```
Comment (shewa12) [inline templates/course/reviews.php:152]: Could you please explain why esc_attr function removed?

File: templates/course/reviews.php [hunk 2]
```diff
@@ -145,24 +148,26 @@
...
-					<input type="hidden" name="review_id" value="<?php echo esc_attr( $my_rating ? $my_rating->comment_ID : '' ); ?>"/>
```
Comment (shewa12) [inline templates/course/reviews.php:153]: Why esc_attr function has removed?

File: templates/course/reviews.php [hunk 3]
```diff
@@ -145,24 +148,26 @@
...
-						?>
+						<div class="tutor-ratings tutor-ratings-lg tutor-ratings-selectable" tutor-ratings-selectable>
+							<?php
+								tutor_utils()->star_rating_generator( tutor_utils()->get_rating_value( $my_rating ? $my_rating->rating : 0 ) );
+							?>
+						</div>
 					</div>
 					<div class="tutor-form-group">
-						<textarea name="review" placeholder="<?php esc_html_e( 'write a review', 'tutor-lms-elementor-addons' ); ?>"><?php echo esc_textarea( $my_rating ? $my_rating->comment_content : '' ); ?></textarea>
+						<textarea name="review" placeholder="<?php _e( 'write a review', 'tutor-lms-elementor-addons' ); ?>"><?php echo stripslashes( $my_rating ? $my_rating->comment_content : '' ); ?></textarea>
```
Comment (shewa12) [inline templates/course/reviews.php]: stripslashes should have used inside esc_textarea function like: esc_textarea( stripslashes( $var ) );

**Review by shewa12 — CHANGES_REQUESTED** [review]
Please review the mention comments

File: templates/course/reviews.php (hunk 2 above)
Comment (wprashed) [reply templates/course/reviews.php:153]: Bhai I have fixed the issue

File: templates/course/reviews.php (hunk 3 above)
Comment (wprashed) [reply templates/course/reviews.php]: Bhai I have fixed the issue

**Pull request merged**